concurrent/src/main/scala/org/powerscala/concurrent/Time.scala

```diff
@@ -342,7 +342,7 @@ case class Elapsed(time: Double) {
     } else {
       "ms"
     }
-    val s = if (value >= 2 && ending != "ms") "s" else ""
+    val s = if (Math.round(value) != 1 && ending != "ms") "s" else ""
```

Comment:
and since I'm already calling math.round(value) immediately below this, you could extract that into a variable instead of doing it twice.